infra/feast-operator/test/testdata/feast_integration_test_crs/v1alpha1_ephemeral_3_services_pods_featurestore.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: simple-feast-setup
+spec:
+  feastProject: my_project
+  services:
+    onlineStore:
+      image: 'localhost/feastdev/feature-server:dev'
+      persistence:
+        file:
+          path: /data/online_store.db
```

Comment:
i believe these are set by default ... by the operator. no need to specify

Reply:
removed those paths from feast yaml sample file.